.travis.yml

```diff
@@ -1,7 +1,9 @@
+dist: trusty
 language: scala
 scala:
    - 2.10.5
    - 2.11.7
-   - 2.12.0-M3
+   - 2.12.10
+   - 2.13.0
```

Comment:
Shouldn't it be `2.13.1`?

Reply:
I can try that now. Last time I tried, it wasn't available yet.

Reply:
Cool, that worked.